### PR TITLE
Fix value to string in CI

### PR DESCRIPTION
**Pull request by yixinglu**
as title

### CODE REVIEW

File: tests/common/utils.py [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+    return real == expect
+
+
+# Value to String conversation
+def _value_to_string(value):
+    if value.getType() == CommonTtypes.Value.__EMPTY__:
+        return '__EMPTY__'
+    elif value.getType() == CommonTtypes.Value.NVAL:
+        return '__NULL__'
```
Comment (laura-ding) [inline tests/common/utils.py:114]: better print the detailed empty type

File: tests/common/utils.py [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    elif value.getType() == CommonTtypes.Value.DTVAL:
+        return date_time_to_string(value.get_dtVal())
+    elif value.getType() == CommonTtypes.Value.EVAL:
+        return edge_to_string(value.get_eVal())
+    elif value.getType() == CommonTtypes.Value.LVAL:
+        return list_to_string(value.get_lVal())
+    elif value.getType() == CommonTtypes.Value.VVAL:
+        return f'({value.get_vVal().vid})'
+    elif value.getType() == CommonTtypes.Value.UVAL:
+        return list_to_string(value.get_uVal())
```
Comment (laura-ding) [inline tests/common/utils.py:136]: set_to_string

File: tests/common/utils.py [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+    elif type(value) is CommonTtypes.Step:
+        return step_to_string(value)
+    elif type(value) is list:
+        return list_to_string(value)
+    elif type(value) is dict:
+        return map_to_string(value)
+    elif type(value) is set:
+        return set_to_string(value)
+    else:
+        return value.__repr__()
```
Comment (laura-ding) [inline tests/common/utils.py:163]: str(value)

File: tests/common/utils.py [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+def date_to_string(date):
+    return f'{date.year}/{date.month}/{date.day}'
+
+
+def time_to_string(time):
+    return f'{time.hour}:{time.minute}:{time.sec}.{time.microsec}'
+
+
+def date_time_to_string(date_time):
+    return f'{date_to_string(date_time)} {time_to_string(date_time)}'
```
Comment (laura-ding) [inline tests/common/utils.py:181]: add `T` between date and time

File: tests/common/utils.py [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+    elif isinstance(col, PathVal):
+        value.set_pVal(col.to_value())
+    else:
+        raise ValueError(f'Wrong val type: {str(col)}')
+    return value
+
+
+def find_in_rows(row, rows):
+    for r in rows:
+        assert len(r.values) == len(row)
```
Comment (laura-ding) [inline tests/common/utils.py:297]: print row and rows

File: tests/common/utils.py [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+    elif isinstance(col, CommonTtypes.Path):
+        value.set_pVal(col)
+    elif isinstance(col, PathVal):
+        value.set_pVal(col.to_value())
+    else:
+        raise ValueError(f'Wrong val type: {str(col)}')
+    return value
+
+
+def find_in_rows(row, rows):
```
Comment (laura-ding) [inline tests/common/utils.py:295]: where use it?

File: tests/common/path_value.py [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+
+class PathVal:
+    def __init__(self, items):
+        self.items = items
+
+    def to_value(self):
+        path = CommonTtypes.Path()
+        path.steps = []
+        for col, j in zip(self.items, range(len(self.items))):
```
Comment (laura-ding) [inline tests/common/path_value.py:18]: ditto

File: tests/common/path_value.py [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# --coding:utf-8--
+#
+# Copyright (c) 2019 vesoft inc. All rights reserved.
```
Comment (laura-ding) [inline tests/common/path_value.py:3]: 2020

File: tests/common/path_value.py [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+from nebula2.common import ttypes as CommonTtypes
+
+
+class PathVal:
+    def __init__(self, items):
+        self.items = items
```
Comment (laura-ding) [inline tests/common/path_value.py:13]: check the `items`'s format

File: tests/common/nebula_test_suite.py [hunk 10]
```diff
@@ -195,316 +201,96 @@ def check_resp_failed(self, resp, error_code: ttypes.ErrorCode = ttypes.ErrorCod
...
-        if resp.data is None:
-            return False, 'resp.data is None'
-        rows = resp.data.rows
+        assert resp.data is not None, 'resp.data is None'
 
-        msg = 'len(rows)[%d] < len(expect)[%d]' % (len(rows), len(expect))
-        if len(rows) < len(expect):
-            return False, msg
+        rows = resp.data.rows
+        assert len(rows) >= len(expect)
```
Comment (laura-ding) [inline tests/common/nebula_test_suite.py:220]: here can print the rows and expect

File: tests/common/utils.py (hunk 1 above)
Comment (yixinglu) [reply tests/common/utils.py]: fixed.

File: tests/common/utils.py (hunk 2 above)
Comment (yixinglu) [reply tests/common/utils.py]: fixed.

File: tests/common/utils.py (hunk 3 above)
Comment (yixinglu) [reply tests/common/utils.py]: fixed.

File: tests/common/utils.py (hunk 4 above)
Comment (yixinglu) [reply tests/common/utils.py]: fixed.

File: tests/common/utils.py (hunk 5 above)
Comment (yixinglu) [reply tests/common/utils.py]: fixed.

File: tests/common/utils.py (hunk 6 above)
Comment (yixinglu) [reply tests/common/utils.py:296]: used in nebula_test_suite.py

File: tests/common/path_value.py (hunk 7 above)
Comment (yixinglu) [reply tests/common/path_value.py]: fixed.

File: tests/common/path_value.py (hunk 8 above)
Comment (yixinglu) [reply tests/common/path_value.py]: fixed.

File: tests/common/nebula_test_suite.py (hunk 10 above)
Comment (yixinglu) [reply tests/common/nebula_test_suite.py]: fixed it.

File: tests/common/path_value.py (hunk 9 above)
Comment (yixinglu) [reply tests/common/path_value.py:13]: fixed.

**Review by laura-ding — APPROVED** [review]

**Review by jievince — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Pull request merged**